sites.json

```diff
@@ -1478,7 +1489,7 @@
 
     {
         "name": "Dwolla",
-        "url": "http://help.dwolla.com/customer/portal/articles/269223-how-can-i-delete-my-dwolla-account-",
+        "url": "http://help.dwolla.com/customer/en/portal/articles/1203269-closing-your-dwolla-account",
```

Comment:
Like with Credit Karma, this is better as the direct deactivation link: http://www.dwolla.com/deactivate